hc/api/tests/test_healthwealth.py

```diff
@@ -0,0 +1,48 @@
+from hc.api.models import ExternalChecks, Check
```

Comment:
good approach in implementing the tests for the integration.

Reply:
Thanks